packages/website/components/Layout.tsx

```diff
@@ -12,6 +12,7 @@ const styles = {
     overflow: 'hidden',
     gridTemplateAreas: `"topbar topbar"
     "sidemenu content"`,
+    gridTemplateColumns: '2fr 10fr',
```

Comment:
We are using the `Grid` component here, so I would say should remove this from the css class and update the prop in line 39

```jsx
<Grid
      className={styles.grid}
      columns="2fr 10fr"
      rows="auto 1fr"
      columnGap="none"
    >
```